shap/benchmark/metrics.py

```diff
@@ -73,298 +74,298 @@ def score_function(X_train, X_test, y_train, y_test, attr_function, trained_mode
 
-def remove_absolute_impute__r2(X, y, model_generator, method_name, num_fcounts=11):
+def remove_absolute_impute__r2(X, y, model_generator, explainer, num_fcounts=11):
     """ Remove Absolute (impute)
     xlabel = "Max fraction of features removed"
     ylabel = "1 - R^2"
     transform = "one_minus"
     sort_order = 9
     """
-    return __run_measure(measures.remove_impute, X, y, model_generator, method_name, 0, num_fcounts, sklearn.metrics.r2_score)
+    return __run_measure(measures.remove_impute, X, y, model_generator, explainer, 0, num_fcounts, sklearn.metrics.r2_score)
 
-def remove_absolute_impute__roc_auc(X, y, model_generator, method_name, num_fcounts=11):
+def remove_absolute_impute__roc_auc(X, y, model_generator, explainer, num_fcounts=11):
     """ Remove Absolute (impute)
     xlabel = "Max fraction of features removed"
     ylabel = "1 - ROC AUC"
     transform = "one_minus"
     sort_order = 9
     """
-    return __run_measure(measures.remove_mask, X, y, model_generator, method_name, 0, num_fcounts, sklearn.metrics.roc_auc_score)
-
-def keep_positive_retrain(X, y, model_generator, method_name, num_fcounts=11):
-    """ Keep Positive (retrain)
-    xlabel = "Max fraction of features kept"
-    ylabel = "Mean model output"
-    transform = "identity"
-    sort_order = 6
-    """
-    return __run_measure(measures.keep_retrain, X, y, model_generator, method_name, 1, num_fcounts, __mean_pred)
-
-def keep_negative_retrain(X, y, model_generator, method_name, num_fcounts=11):
-    """ Keep Negative (retrain)
-    xlabel = "Max fraction of features kept"
-    ylabel = "Negative mean model output"
-    transform = "negate"
-    sort_order = 7
-    """
-    return __run_measure(measures.keep_retrain, X, y, model_generator, method_name, -1, num_fcounts, __mean_pred)
-
-def remove_positive_retrain(X, y, model_generator, method_name, num_fcounts=11):
-    """ Remove Positive (retrain)
-    xlabel = "Max fraction of features removed"
-    ylabel = "Negative mean model output"
-    transform = "negate"
-    sort_order = 11
-    """
-    return __run_measure(measures.remove_retrain, X, y, model_generator, method_name, 1, num_fcounts, __mean_pred)
-
-def remove_negative_retrain(X, y, model_generator, method_name, num_fcounts=11):
-    """ Remove Negative (retrain)
-    xlabel = "Max fraction of features removed"
-    ylabel = "Mean model output"
-    transform = "identity"
-    sort_order = 12
-    """
-    return __run_measure(measures.remove_retrain, X, y, model_generator, method_name, -1, num_fcounts, __mean_pred)
-
-def __run_measure(measure, X, y, model_generator, method_name, attribution_sign, num_fcounts, summary_function):
+    return __run_measure(measures.remove_mask, X, y, model_generator, explainer, 0, num_fcounts, sklearn.metrics.roc_auc_score)
+
+# def keep_positive_retrain(X, y, model_generator, method_name, num_fcounts=11):
```

Comment:
remove commented code?

Reply:
This was originally commented out because we would not be implementing them for the new API. However, all old APIs will be cleaned up together later once the new API can completely replace the old one.